armi/reactor/blueprints/isotopicOptions.py

```diff
@@ -524,7 +509,7 @@ def autoSelectElementsToKeepFromSettings(cs):
                 )
             )
 
-    elif cs["xsKernel"] in ["SERPENT", "MC2v3", "MC2v3-PARTISN"]:
+    elif cs["xsKernel"] in ["", "SERPENT", "MC2v3", "MC2v3-PARTISN"]:
```

Comment:
Yikes. Maybe this should be a plugin thing? No need to do it now though

Reply:
Absolutely. There are commends above in the docstring from earlier stating that this needs to be migrated into the plugins. The behavior is surprising and complicated here.